lib/index.js

```diff
@@ -282,7 +287,7 @@ api.createIndexes = brCallbackify(async options => {
  * @return the new GridFSBucket instance
  */
 api.createGridFSBucket = options => {
-  const opts = Object.assign({}, {writeConcern: api.writeOptions}, options);
+  const opts = Object.assign({}, {...api.writeOptions}, options);
```

Comment:
```suggestion
  const opts = {...api.writeOptions, ...options);
```

Reply:
I believe we want out `writeOptions` to layer over the other ones correct?

```js
var a = {foo: true}
undefined
var b = {foo: false}
undefined
var c = {...a, ...b}
undefined
c
{foo: false}
```
so I think we want this to be:

```js
{...options, ...api.writeOptions}
```

so that the writeConcern layers over the the one passed into to `createGridFSBucket` or am I not getting something here?

made the changes here: https://github.com/digitalbazaar/bedrock-mongodb/pull/65/commits/163a951857899ac70ee2944a892eb1b6eda2a484

Reply:
No, we want the caller of the API to be able to override the defaults that come from `api.writeOptions`.